api/serverpb/server.proto

```diff
@@ -141,6 +141,16 @@ message SlackAlertingSettings {
   ];
 }
 
+// STTCheckIntervals represents intervals between STT checks.
+message STTCheckIntervals {
+  // Default check interval
+  google.protobuf.Duration default_interval = 1;
```

Comment:
In ticket it has name `standard interval`

Reply:
Done